package.json

```diff
@@ -11,9 +11,11 @@
   "files": [
     "index.js"
   ],
-  "dependencies": {},
   "devDependencies": {
-    "tape": "~5.5.3"
+    "@ljharb/eslint-config": "^21.0.0",
+    "aud": "^2.0.0",
+    "eslint": "=8.8.0",
```

Comment:
is the `=` intentional?

Reply:
Yep; my eslint config requires it be pinned since eslint 8.9+ broke parsing strict mode in ES3.

I’m not stuck on my personal config ofc, it’s just what i set up. Happy to change it to whatever you prefer.

Reply:
I hadn't seen this syntax before, I thought pinned would just be `"eslint": "8.8.0"`. Happy to keep it like this 👍

Reply:
Those are equivalent, but the explicit `=` conveys intention better to me :-)